genotype/store/export.py

```diff
@@ -56,7 +56,7 @@ def _get_equality(analysis_1: dict, analysis_2: dict) -> dict:
     compare_dict = {}
 
     for snp in set(list(analysis_1.keys()) + list(analysis_2.keys())):
-        if analysis_1.get(snp) == analysis_2.get(snp):
+        if set(analysis_1.get(snp)) == set(analysis_2.get(snp)):
```

Comment:
any chance that the ```snp``` is a key in one dict, but not in the other? if so the ```.get``` method would by default return None for the key where it does not exist, and the ```set```function would perhaps raise an exception. Maybe we could ad the extra argument ```[]```to the get function, returning an empty list by default

```suggestion
        if set(analysis_1.get(snp, [])) == set(analysis_2.get(snp, [])):
```